pkg/sdk/testint/external_functions_integration_test.go

```diff
@@ -125,6 +124,7 @@ func TestInt_ExternalFunctions(t *testing.T) {
 
 	t.Run("alter external function: set api integration", func(t *testing.T) {
 		e := createExternalFunction(t)
+		e.ID()
```

Comment:
I'm guessing this should be assigned to `id` below

Reply:
oh, I missed that one. Actually, no, it should be just removed. I left `NewSchemaObjectIdentifierWithArguments` for the identifiers rework (because of the arguments part and the fact that `NewSchemaObjectIdentifierWithArguments` is not equal to `NewSchemaObjectIdentifier` - because of the lacking arguments underneath).

I will remove this e.ID() in the following PR, it's not hurting

Reply:
(I have added this topic to the identifiers rework)